ide/lsp.client/src/org/netbeans/modules/lsp/client/debugger/Utils.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.netbeans.modules.lsp.client.debugger;
+
+import java.util.logging.Logger;
+import javax.swing.SwingUtilities;
+
+import org.openide.text.Annotatable;
+import org.openide.text.Line;
+import org.openide.text.Line.ShowOpenType;
+import org.openide.text.Line.ShowVisibilityType;
+
+public final class Utils {
+
+    private static final Logger logger = Logger.getLogger(Utils.class.getName());
+
+    private static Object currentLine;
+
+    private Utils() {
+    }
+
+    static synchronized void markCurrent (final Object line) {
+        unmarkCurrent ();
+
+        Annotatable[] annotatables = (Annotatable[]) line;
+        int i = 0, k = annotatables.length;
+
+        // first line with icon in gutter
+        DebuggerAnnotation[] annotations = new DebuggerAnnotation [k];
+        if (annotatables [i] instanceof Line.Part)
```

Comment:
I see, that the block braces are not strictly needed here, but would make it easier to read (same is true for the if and for below. This looks like Python style, which we are not).

Reply:
Result of copy and paste. Made different from original in 2d326bd9f8